### ISSUE TITLE
Organization required for create application

### ISSUE THREAD

**Issue by willtome** [ansible/tower-cli#652]
The organization field is required to create an application but `--organization` is not a valid option. 

```
tower-cli application create --user tower-api --name 'Tower API' --client-type confidential --authorization-grant-type password
```
```
Error: The Tower server claims it was sent a bad request.

POST https://tower34/api/v2/applications/
Params: None
Data: {"name": "Tower API", "client_type": "confidential", "user": 2, "authorization_grant_type": "password"}

Response: {"organization":["This field is required."]}
```

Tower 3.4

**Comment by jainnikhil30** [comment]
@willtome 

Can you please try this with latest tower-cli (3.3.2). I am not able to reproduce this.

```
[niks@localhost ~]$ tower-cli application create --name 'Tower API' --client-type confidential --authorization-grant-type password --organization Default
*** DETAILS: The organization field is given as a name; looking it up. ********
*** DETAILS: Getting the record. **********************************************
GET https://52.77.209.20/api/v2/organizations/
Params: [('name', 'Default')]

*** DETAILS: Checking for an existing record. *********************************
GET https://52.77.209.20/api/v2/applications/
Params: [('name', u'Tower API')]

*** DETAILS: Writing the record. **********************************************
POST https://52.77.209.20/api/v2/applications/
Data: {'name': u'Tower API', 'client_type': 'confidential', 'organization': 1, 'authorization_grant_type': 'password'}

Resource changed.
== ========= ============ ============= ======================== ================== ============ 
id   name    client_type  redirect_uris authorization_grant_type skip_authorization organization 
== ========= ============ ============= ======================== ================== ============ 
 1 Tower API confidential               password                              false            1
== ========= ============ ============= ======================== ================== ============

**Comment by willtome** [comment]
Looks like it may have been fixed in PR #547 for 3.3.1. Just updated to 3.3.2 and  it appears to be working correctly.
